addons/mrp/static/src/product_catalog/order_line.js

```diff
@@ -0,0 +1,8 @@
+import { ProductCatalogOrderLine } from "@product/product_catalog/order_line/order_line";
+import { patch } from "@web/core/utils/patch";
+
+patch(ProductCatalogOrderLine.prototype, {
+    get showPrice() {
+        return super.showPrice && this.env.orderResModel !== "mrp.production";
+    }
+});
```

Comment:
Don't forget new Line at the end of each file
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file